apps/experiments/migrations/0120_alter_participant_unique_together_and_more.py

```
@@ -0,0 +1,48 @@
+            name='participantdata',
+            unique_together=None,
+        ),
+        migrations.RemoveField(
+            model_name='participantdata',
+            name='experiment',
+        ),
+        migrations.RemoveField(
+            model_name='participantdata',
+            name='team',
+        ),
+    ]
```

Ruff (0.14.5)

Mutable class attributes should be annotated with `typing.ClassVar`

(RUF012)